docs/extensions.rst

```diff
@@ -213,3 +213,32 @@ Keys:
 
 The ASGI server will only send the trailing headers in case the client has sent the
 ``TE: trailers`` header in the request.
+
+Debug
+-----
+
+The debug extension allows a way to send debug information from the ASGI application to
+an entity acting as an ASGI server. This extension is not meant to be used in production,
```

Comment:
Can we improve the "entity acting as an ASGI server"?

Reply:
I'd just drop it and just say "from an ASGI application in its responses".